first-analysis-steps/analysisflow.md

```diff
@@ -0,0 +1,114 @@
+This list is by no means exhaustive, so if there are any other tools you use often, please, feel free to add them by contributing to this lesson. 
+
+* [ROOT](https://root.cern/) and its python twin sister pyROOT are open-source data analysis frameworks used extensively in HEP community. 
+* [Scikit-HEP project](https://github.com/scikit-hep), a community project for a Python-based Big Data analysis ecosystem for Particle Physics. Consists of multiple complementary and interoperable Python modules, some of them are listed here:
+   - [uproot](https://github.com/scikit-hep/uproot4) a python module that allows to read the root files (that are the output of the DaVinci package). 
+   - [awkward-array](https://github.com/scikit-hep/awkward-1.0) a Python module for a friendly manipulation of data structures common in Particle Physics, i.e., nested, variable-sized data, including arbitrary-length lists, records, mixed types, and missing data, using NumPy-like idioms.
+   - [boost histogram](https://github.com/scikit-hep/boost-histogram) a python version of the C++14 library [Boost::Histogram](https://www.boost.org/doc/libs/1_77_0/libs/histogram/doc/html/index.html) for histogramming. 
+   - [iminuit](https://github.com/scikit-hep/iminuit) a jupyter-friendly python interface for minimization package [Minuit2](https://root.cern.ch/doc/master/Minuit2Page.html).
+   - [mplhep](https://github.com/scikit-hep/mplhep) a set of high energy physics helpers for [matplotlib](https://matplotlib.org/).
+   - etc.
+* [RooFit](https://root.cern/manual/roofit/) a fitting, fit model defining, plotting and "toy model" generation package supported by CERN ROOT team. Can be used in both C++ and python.
+* [zfit](https://github.com/zfit/zfit) a fitting and fit model defining Python package well integrated withthe [Scikit-HEP project](https://github.com/scikit-hep) analysis ecosystem.
+
+
+{% endcallout %}
+
+Discussions on the new analysis tools that might be useful for the LHCb community are held in the [Work Package 4](https://lhcb-dpa.web.cern.ch/lhcb-dpa/wp4/index.html) of the Data Processing & Analysis project (DPA). 
+
+### Analysis Preservation
+
+When the samples are ready one can proceed with developing the necessary macros and scripts to perform the analysis steps, like applying additional selections, fitting distributions, computing efficiencies and acceptances, etc. 
+Starting from the ntuples a typical analysis will consist of the following steps: 
+
+1. Defining and applying selections, including kinematic, particle identification, multivariate analysis, signal/background separations by fitting.
+2. Computing and applying callibrations and corrections. 
+Some of the calibrations, like [TrackCalib](https://gitlab.cern.ch/lhcb/Urania/tree/master/TrackCalib) and [PIDCalib2](https://gitlab.cern.ch/lhcb-rta/pidcalib2), are calibrated on the "standard candle" decays and can be used by the users directly. 
```

Comment:
What do you mean? Rather calibration samples?

Reply:
Yes, I was thinking about how to explain it to a person with a very general knowledge of physics and I thought that the "standard candle" is sort of a universal term, but if you think it is even more confusing...

Reply:
I think "standard candle" is indeed very confusing. I would use "calibration sample" and say that calibration samples are usually decays that can be reconstructed easily, are well understood and all together representative for most of the physics programme of LHCb. Maybe add that they are mainly used for tracking efficiency and PID corrections

Reply:
TrackCalib and PIDCalib are examples of calibrations via  tag-and-probe approach, not a standard candle. A standard candle approach would be using a decay with a very well known branching fraction as efficiency normalisation. (think about the role of the J/psi -> mu mu / J/psi -> e e in R(K), and D -> K3pi / D -> Kpi)

Reply:
Thanks for clarification @laurentdufour  and comments @eduardo-rodrigues and @lukascalle. I have fixed it in the most generic way to not confuse the students more.